sycl/test/basic_tests/buffer/buffer.cpp

```diff
@@ -403,8 +403,29 @@ int main() {
             range<1>{3}, [=](id<1> index) { B[index] = 20; });
       });
     }
-    // Data is not copied back in the desctruction of the buffer created from
-    // pair of iterators
+    // Data is copied back in the desctruction of the buffer created from
+    // pair of non-const iterators
+    for (int i = 0; i < 2; i++)
+      assert(data1[i] == -1);
+    for (int i = 2; i < 5; i++)
+      assert(data1[i] == 20);
+    for (int i = 5; i < 10; i++)
+      assert(data1[i] == -1);
+  }
+
+  // Check that data is not copied back in the desctruction of the buffer
+  // created from pair of const iterators
+  {
+    std::vector<int> data1(10, -1);
+    {
+      buffer<int, 1> b(data1.cbegin() + 2, data1.cbegin() + 5);
```

Comment:
`{}`

Reply:
Hmm, It's really big test and in all buffer constructions `()` is used, I think this change also should be done in separate patch.

Reply:
Yes you are right. It is cosmetic change.